networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -0,0 +1,120 @@
+[id="configuring-additional-network"]
+= Configuring an additional network
+include::modules/common-attributes.adoc[]
+:context: configuring-additional-network
+
+toc::[]
+
+As a cluster administrator, you can configure an additional network for your cluster. The following network types are supported:
+
+* xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-bridge-object_configuring-additional-network[Bridge]
+* xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-host-device-object_configuring-additional-network[Host device]
+* xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-ipvlan-object_configuring-additional-network[IPVLAN]
+* xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-macvlan-object_configuring-additional-network[MACVLAN]
+
+== Approaches to managing an additional network
+
+You can manage the life cycle of an additional network by two approaches. Each approach is mutually exclusive and you can only use one approach for managing an additional network at a time. For either approach, the additional network is managed by a Container Network Interface (CNI) plug-in that you configure.
+
+For an additional network, IP addresses are provisioned through an IPAM CNI plug-in that you configure as part of the additional network. The IPAM plug-in supports a variety of IP address assignment approaches including DHCP and static assignment.
```

Comment:
Do we want to define that IPAM is IP Address Management here? (maybe it's elsewhere...)